### ISSUE TITLE
risky comment lines?

### ISSUE THREAD

**Issue by alexv66** [PHPOffice/PHPExcel#1132]
hello

thank you for good library, but recently client complained about these comments.
It's not critical, but could you rephrase it  a bit ( without "Hacked ") ?

> `Both` of these comments almost got my site suspended because my hosting company thought they were malicious hacks. Please change wording?
> 
> PHPExcel/Shared/OLERead.php: // Hacked by Andreas Rehm 2006 to ensure correct result of the <<24 block on 32 and 64bit systems
> 
>PHPExcel/Reader/Excel5.php: // Hacked by Andreas Rehm 2006 to ensure correct result of the <<24 block on 32 and 64bit systems

**Comment by PowerKiKi** [comment]
Done in https://github.com/PHPOffice/PhpSpreadsheet/commit/fb1d280e825381c97acd62e5e4292bea47e15785
